orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltOrchestrator.java

```diff
@@ -949,7 +949,7 @@ public void stopClusterManagerAgent(GatewayConfig gatewayConfig, Set<Node> allNo
                 Set<String> allHostnames = responsiveNodesUnderStopping.stream().map(Node::getHostname).collect(Collectors.toSet());
                 runSyncAll(sc, allHostnames, responsiveNodesUnderStopping, exitCriteriaModel);
 
-                refreshPillars(gatewayConfig, responsiveNodes, exitCriteriaModel, sc);
+                refreshPillars(gatewayConfig, allNodes, exitCriteriaModel, sc);
```

Comment:
why was this change necessary? could you add the explanation to the commit message?

Reply:
It is resolved by @lturcsanyi, so I will rebase to the latest master and it will disappear